data/jibauni/jibauni missions.txt

```diff
@@ -0,0 +1,579 @@
+				goto choice
+			label rebelsgood
+			action
+				set "jibauni: dran-lin: says rebels are good"
+			`	"I am... surprised that you say that. Most I have met give them a bad reputation and a good scolding. But I would be happy to be taken!"`
+			label choice
+			choice
+				`	"Alright, I'll take you."`
+					accept
+				`	"Anyway, I need to do something else, so I'm not going to take you after all."`
+			label decline
+			`	The Jibauni drifts off disappointedly.`
+				decline
+	on complete
+		payment
+		event "jibauni: dran-lin gone" 1
+		conversation
+			`Dran-lin says, "I am going to explore the spaceport. Could you meet me at the main hotel in a few hours? If you do not, I will assume you do not want to take me back."`
+
+event "jibauni: dran-lin gone"
+
+mission "Jibauni: Dran-lin 1"
+	priority
+	name "Take Dran-lin to <planet>"
+	description "Dran-lin didn't love the way <origin> felt, and would like you to take him to <destination> to see if it was just <origin> and not all Rebel planets that were like that."
+	passengers 1
+	source
+		government "Jibauni (Rebel)"
+	destination
+		government "Jibauni (Rebel)"
+		distance 1 1
+	to offer
+		has "Jibauni: Dran-lin 0: done"
+		not "event: jibauni: dran-lin gone"
+	on offer
+		conversation
+			`Dran-lin meets you at the main hotel, which happens to be called Jivam Dar-li. "Could you take me to <destination>? I do not get a good sense from the spaceport of <origin>. Perhaps this city in particular is just bad."`
+			choice
+				`	"Alright."`
+				`	"Nope, sorry. I agreed to take you... once. No more."`
+					goto declining?
+			`	"Perfect! I will be ready when you are."`
+				accept
+			label declining?
+			`	"You can not do that! It is not very kind; there is unlikely to be anybody who will take me back to Jibauni space. Please take me!"`
+			choice
+				`	"Oh, whatever. I'll take you to <planet>."`
+					accept
+				`	"My word is my word. Goodbye."`
+			action
+				"reputation: Jibauni" -= 3
+			`	Dran-lin continues to protest, until you leave, slamming the door behind you. The Jibauni may or may not be happy about this.`
+				decline
+	on complete
+		conversation
+			`Dran-lin leaves the <ship> for a scouting run. "I will be back soon," he promises.`
+			branch "dran-lin put in jail"
+				random < 3
+			choice
+				`	(Wait.)`
+			branch "dran-lin likes"
+				random < 60
+				not "jibauni: dran-lin: says rebels are good"
+			branch "dran-lin likes with input"
+				random < 50
+				has "jibauni: dran-lin: says rebels are good"
+			label "dran-lin dislikes"
+			action
+				event "jibauni: dran-lin abandoned" 1
+			`	He is back soon. "I really do not like this place," he announces. "It is even worse than <origin>, and that is saying a lot. Please meet me in the spaceport at the Indon-alo in an hour. I am going to the capital to ask a few questions. I mean, I guess I can stay if you do not come, but it would really be much appreciated if you do."`
```

Comment:
```suggestion
			`	Dran-lin returns after only brief span of time. "I really do not like this place," he announces. "It is even worse than <origin>, and that is saying a lot. Please meet me in the spaceport at the Indon-alo in an hour. I am going to the capital to ask a few questions. I mean, I guess I can stay if you do not come, but it would really be much appreciated if you do."`
```